### DISCUSSION TITLE
TuyaV2 wird mir nicht angezeigt + Energie Monitor

### DISCUSSION THREAD

**Discussion by Captonik** [tuya/tuya-home-assistant#157] [Q&A]
Guten Tag,

leider wird mir bei Integration Hinzufügen kein Tuya_V2 angezeigt richtig installiert ist dieses.

Zweite Frage kommt eine Integration der Energieüberwachung sodass ich den Stromverbrauch Aktuell und gesamt auf die tage gesehen messen kann?

**Comment by tbratfisch** [comment]
@Captonik: please post in in English, you cannot expect that the devs are able to understand German
Now in German: bitte schreibe in englisch, du kannst nicht erwarten dass deutsch hier verstanden wird - falls du kein englisch sprichst suche dir bitte jemanden der dir das hier uebersetzt
